test/src/nyc-test.js

```diff
@@ -211,6 +248,42 @@ describe('nyc', function () {
       })
     })
 
+    describe('compile handlers for custom extensions are assigned', function () {
+      it('assigns a function to custom extensions', function () {
+        var nyc = new NYC({
+          cwd: path.resolve(__dirname, '../fixtures')
```

Comment:
I changed the `cwd` to use the fixtures `package.json` for these two tests, so that the looping can be tested as well as the case-insensitive matching in the next one.